docs/cloud/managing-airbyte-cloud/manage-schema-changes.md

```diff
@@ -1,12 +1,27 @@
 # Manage schema changes
 
-Once every 24 hours, Airbyte checks for changes in your source schema and allows you to review the changes and fix breaking changes. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
+You can specify for each connection how Airbyte should handle any change of schema in the source. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
 
-:::note 
+Airbyte checks for any changes in your source schema before every sync or once every 24 hours, whichever is more frequent.
```

Comment:
@mfsiega-airbyte I think I recall this being the case, but am not sure now if this is true. Is it still once every 24 hours or does it check on sync start as well?

Reply:
Updated to say every 24 hours

Reply:
I would phrase it as: `Airbyte checks for any changes in your source schema before syncing, at most once every 24 hours.`

Reply:
Great, updated 👍